.takt/facets/instructions/generate-design.md

```
@@ -0,0 +1,46 @@
+   - 要件トレーサビリティ表
+   - エラー処理戦略
+   - テスト戦略
+8. 成果物を保存する
+
+**成果物の保存先:**
+- `.kiro/specs/{feature}/design.md`
+- `.kiro/specs/{feature}/research.md`
+
+**必須出力（見出しを含める）**
+
+## 概要
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)